src/main/java/co/cask/http/AbstractHttpResponder.java

```diff
@@ -53,6 +54,11 @@ public void sendJson(HttpResponseStatus status, Object object, Type type) {
   @Override
   public void sendJson(HttpResponseStatus status, Object object, Type type, Gson gson) {
     try {
+      if (object instanceof Iterable) {
```

Comment:
This is bad. I'm not sure what would be a better solution though. @chtyim: thoughts?